kbase-extension/static/kbase/js/widgets/kbaseTabs.js

```diff
@@ -2,54 +2,59 @@
 
     Easy widget to serve as a tabbed container.
 
+    It is an extension of the Bootstrap 3 tab (https://getbootstrap.com/docs/3.4/javascript/#tabs).
+
     var $tabs =  new kbaseTabs($('#tabs'), {
-            tabPosition : 'bottom', //or left or right or top. Defaults to 'top'
-            canDelete : true,       //whether or not the tab can be removed. Defaults to false.
+            canDelete : true,       // boolean; whether a tab which does not otherwise specify this option
```

Comment:
do you need to add `confirmDelete` here too?

Reply:
I'm adding it in, although my guess is that either the docs were created before the confirmDelete option was added, or this specific example has canDelete set to false, in which case confirmDelete is not necessary.